packages/shared-state-bat_links_info/files/etc/uci-defaults/shared-state_bat_links_info_cron

```diff
@@ -4,10 +4,18 @@ unique_append()
 {
 	grep -qF "$1" "$2" || echo "$1" >> "$2"
 }
```

Comment:
This function is not used anymore should be removed, also the script should be renamed ;)